pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[tool.poetry]
+name = "peregrine"
+version = "3.0.1"
+description = "Query interface to get insights into data in Gen3 Commons"
+authors = ["CTDS UChicago <cdis@uchicago.edu>"]
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/uc-cdis/peregrine"
+include = [
+  "NOTICE",
+]
+
+[tool.poetry.dependencies]
+python = "^3.6"
+authutils = "^3.1"
+datamodelutils = "^1.0"
+defusedxml = "^0.5"
+dictionaryutils = "^3.0"
+gen3dictionary = "^2.0"
+simplejson = "^3.8"
+stopit = "^1.1"
+Flask = "^1.1"
+Flask-Cors = "^1.9"
+gen3authz = "^0.2"
+PyYAML = "^5.1"
+boto = "^2.36"
+elasticsearch = "^1.2"
+dicttoxml = "^1.5"
+SQLAlchemy = "1.3.5"
+graphene = "^2.0"
+promise = "^2.2"
+Flask-SQLAlchemy-Session = "^1.1"
+psqlgraph = "^3.0"
+gen3datamodel = "^3.0"
+cdispyutils = "^1.0"
+cdiserrors = "^0.1"
+cdislogging = "^1.0"
+uvicorn = "^0.11.2"
+gunicorn = "^20.0.4"
+setuptools = "^56.0.0"
+uvloop = "0.14.0"
+
+[tool.poetry.dev-dependencies]
+pytest = "^6.0"
+python-Levenshtein = "^0.12"
+fuzzywuzzy = "^0.6"
+lxml = "^4.4"
+mock = "^1.0"
+pytest-flask = "^0.15"
+pytest-cov = "^2.8"
+Envelopes = "^0.4"
+indexclient = {git = "https://git@github.com/uc-cdis/indexclient.git", rev = "2.0.0"}
+sheepdog = {git = "https://git@github.com/uc-cdis/sheepdog.git", rev = "chore/python3"}
```

Comment:
Are we referencing a sheepdog branch here? Is that ok?

Reply:
Fixed. Good catch!